easybuild/easyconfigs/p/pytest/pytest-8.2.2-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,76 @@
+easyblock = 'PythonBundle'
+
+name = 'pytest'
+version = '8.2.2'
+
+homepage = 'https://docs.pytest.org/en/latest/'
+description = """The pytest framework makes it easy to write small,
+readable tests, and can scale to support complex functional testing for
+applications and libraries."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('hatchling', '1.18.0'),
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('Python-bundle-PyPI', '2023.10'),
```

Comment:
We should try and avoid depending on `Python-bundle-PyPI` here, see changes made in #21004